.github/workflows/generate_api_docs.yaml

```
@@ -19,6 +19,11 @@ jobs:
       run:
         shell: bash -el {0}
     steps:
+      - name: Install jq
```

```suggestion
      - name: "Install jq 📦"
```
Nit for formatting consistency